client/home/lib/myteam/components/hometeamteammates/container.coffee

```diff
@@ -26,7 +26,7 @@ module.exports = class HomeTeamTeamMatesContainer extends React.Component
   componentWillMount: ->
 
     options =
-      limit : 10
+      limit : 20
```

Comment:
@michaeljackson0802  As it is described in the issue description, initial loads must be for teammates (admins, members) up to 20 members.

With `loadmore` action, you will ask for more members if there are, then already send invitations will be loaded and then disabled members.

Also even if I don't see my disabled members in current teammate section (cause I haven't load all of them), I must be able to see disabled teammates.

These are the current implementations;
for loading teammates
 https://github.com/koding/koding/blob/master/client/app/lib/flux/teams/actions.coffee#L58

for loading already sent invitations
https://github.com/koding/koding/blob/master/client/app/lib/flux/teams/actions.coffee#L87

for loading disabled teammembers
https://github.com/koding/koding/blob/master/client/app/lib/flux/teams/actions.coffee#L240

When you are testing your changes make sure you have enough disabled members (more than 10), already sent invitations (more than 10) and team members (more than 20)

`Filter` and `load more` actions should work as it expected.